.github/workflows/python-package.yml

```diff
@@ -52,5 +52,5 @@ jobs:
     - name: Test nextflow workflow
       run: |
         # Run nextflow tests
-        export NXF_DEFAULT_DSL=1
+        export NXF_DEFAULT_DSL=2
```

Comment:
Unfortunately this Nextflow pipeline is still DSL1 (my bad, I should have picked it up during the hackathon as it's very similar to the submission ones), so you'll have to [fix the Nextflow version <= 22.10](https://github.com/EBIvariation/eva-submission/blob/master/.github/workflows/python-package.yml#L18) for the tests to pass.